packages/flutter/lib/src/services/system_chrome.dart

```diff
@@ -333,6 +333,63 @@ class SystemChrome {
   ///   return /* ... */;
   /// }
   /// ```
+  ///
+  /// For more complex control of the system overlay styles, consider using
+  /// an [AnnotatedRegion] widget instead of calling [setSystemUiOverlayStyle]
+  /// directly. This widget places a value directly into the layer tree where
+  /// it can be hit-tested by the framework. On every frame, the framework will
+  /// select the first annotated region it finds under the status and
+  /// navigation bar and synthesize them into a single style. This process can
+  /// be short-circuted by passing `sized: false` to a single annotated region
```

Comment:
Maybe you can explain more what way you mean by "short-circuited"?  Passing `sized: false` just makes it "hit" if it's on top of the layer tree, regardless of the location passed (At least I think so: is that right?).